dartfn/templates/cloudevent/lib/functions.dart

```diff
@@ -3,11 +3,18 @@ import 'dart:io';
 
 import 'package:functions_framework/functions_framework.dart';
 
-const _encoder = JsonEncoder.withIndent(' ');
+const _encoder = JsonEncoder();
 
 @CloudFunction()
 void function(CloudEvent event, RequestContext context) {
   context.logger
       .info('[CloudEvent] source: ${event.source}, subject: ${event.subject}');
-  stderr.writeln(_encoder.convert(event));
+  stderr.writeln(
+    _encoder.convert(
+      {
+        'message': event,
+        'serverity': LogSeverity.info,
+      },
+    ),
+  );
```

Comment:
And this part improves conformance with https://cloud.google.com/logging/docs/agent/logging/configuration#special-fields

Reply:
sweet!